docs/se/dbclient/01_introduction.adoc

```diff
@@ -16,10 +16,227 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= Helidon DB Client
+= About Helidon DB Client
 :toc:
 :toc-placement: preamble
 :description: Helidon DB Client
 :keywords: helidon, se, database, dbclient
 
-== This page is Under Construction and will be available soon
+
+The Helidon SE DB Client provides a unified, reactive API for working with databases in non-blocking way. 
+
+== Helidon DB Client Features Overview
+
+The DB Client simplfies how you work with databases by allowing the use of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service. Specifically, the Heldion DB Client provides the following:
+
+=== Unified API for data access and query +
+The API was implemented as a layer above JDBC or MongoDB Reactive Streams Java Driver, so any relational databases with JDBC driver or MongoDB are supported.
+
+=== Reactive database access with non-reactive drivers +
+Most JDBC drivers are blocking. Using them in a reactive application is problematic. Helidon DB Client allows the use of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service.
+
+=== System Monitoring +
+The API offers support for health checks, metrics and tracing.
+
+=== Backpressure Management +
+Helidon DB Client performs database operations only when it’s requested by the consumer. This is propagated all the way to the TCP layer.
+
+=== Improved portability between relational database drivers +
```

Comment:
remove 'relational'. The portability is between any drivers (relational or nosql).

Reply:
changed